### PR TITLE
Fix tcpdf config symlink

### PR DESCRIPTION
**Pull request by bezin**
This PR fixes #149.

### CODE REVIEW

File: core-bundle/src/Command/SymlinksCommand.php [hunk 1]
```diff
@@ -116,6 +116,8 @@ protected function executeLocked(InputInterface $input, OutputInterface $output)
     private function generateSymlinks(): void
     {
         $fs = new Filesystem();
+        $uploadPath = $this->getContainer()->getParameter('contao.upload_path');
+        $bundles = $this->getContainer()->getParameter('kernel.bundles');
```
Comment (Toflar) [inline core-bundle/src/Command/SymlinksCommand.php]: Upload path is injected in the constructor, why do you get it here again?
And we could also inject the bundles parameter then :)

File: core-bundle/src/Command/SymlinksCommand.php [hunk 2]
```diff
@@ -173,6 +180,12 @@ private function symlinkThemes(): void
         }
     }
 
+    private function symlinkTcpdf(): void
+    {
+        $coreBundle = new \ReflectionClass($this->bundles['ContaoCoreBundle']);
```
Comment (leofeyer) [inline core-bundle/src/Command/SymlinksCommand.php:184]: How does this work when you have `contao/contao` installed? There is no `ContaoCoreBundle` then, is there?

Comment (bezin) [reply core-bundle/src/Command/SymlinksCommand.php:184]: There is? https://github.com/contao/contao/blob/master/core-bundle/src/ContaoCoreBundle.php

Comment (leofeyer) [reply core-bundle/src/Command/SymlinksCommand.php:184]: I have just realized this as well. 😄

**Comment by leofeyer** [comment]
Thank you @bezin.

**Pull request merged**